Development Environment/Development Environment.md

```diff
@@ -18,21 +18,49 @@ We prefer to keep our development environment as close to "stock" as possible. T
 
 We make use of [GitHub's Swift gitignore](https://github.com/github/gitignore/blob/master/Swift.gitignore) as the default gitignore for our projects. The only change that we typically make is uncommenting the `Pods/` line so that the [`Pods` folder isn't included in source control](../Dependency%20Management/Dependency%20Management.md#checking-in-the-pods-folder).
 
+## Bundler
+
+Many developers naively use the built-in system Ruby to install gems such as Bundler using the `sudo gem install bundler` command. Unfortunately, this can lead to headaches due to the use of `sudo`.
+
+The better way to install Bundler is to get off the system Ruby and use a Ruby version manager to negate the need for `sudo` access when installing or executing gems. Follow the instructions in [Ruby Environment Setup](./Ruby%20Environment%20Setup.md) to install and setup [`rbenv`](https://github.com/rbenv/rbenv).
+
+Once your Ruby is properly setup, installing Bundler is as simple as:
+
+```bash
+gem install Bundler
+```
+
+In order to use bundler in your repo you'll need to create a Gemfile to manage your dependencies. To create that file while in your repo's root directory use:
+
+```bash
+touch Gemfile
+```
+
+Now you can simply add gems to this file using the text editor of your choice.
+
+To install all the gems in your Gemfile simply use the command:
+
+```bash
+bundle install
+```
+
 ## Cocoapods
 
-Many developers naively use the built-in system Ruby to install Cocoapods using the `sudo gem install cocoapods` command. Unfortunately, this can lead to headaches due to the use of `sudo`.
+Once Bundler is properly setup, adding Cocoapods as a dependency is as simple as adding it to your Gemfile:
 
-The better way to install Cocoapods is to get off the system Ruby and use a Ruby version manager to negate the need for `sudo` access when installing or executing gems. Follow the instructions in [Ruby Environment Setup](./Ruby%20Environment%20Setup.md) to install and setup [`rbenv`](https://github.com/rbenv/rbenv).
+```
+gem 'cocoapods'
```

Comment:
Might be worth mentioning the `Gemfile.lock` and it uses for keeping pinning to specific versions of Gems (and it should be committed to the repository)

Reply:
Definitely worth mentioning! We have info about the `Podfile.lock` in the [Dependency Management](https://github.com/BottleRocketStudios/iOS-Project-Standards/blob/master/Dependency%20Management/Dependency%20Management.md#versioning-pods) section. Stuff about the `Gemfile`/`Gemfile.lock` might technically belong there, since it's how you manage your project's "meta" dependencies (e.g. Fastlane, Cocoapods, etc.). In any case, we should probably create some sort of link between these two sections.

Reply:
Updated [Dependency Management](https://github.com/BottleRocketStudios/iOS-Project-Standards/blob/master/Dependency%20Management/Dependency%20Management.md#versioning-pods) to include information on Bundler as well as added a link to more information about dependency management in both the Cocoapods and Bundler sections here.